sklearn/datasets/tests/test_base.py

```diff
@@ -99,9 +99,12 @@ def test_load_files_w_categories_desc_and_encoding(
     test_category_dir_1, test_category_dir_2, load_files_root
 ):
     category = os.path.abspath(test_category_dir_1).split("/").pop()
```

Comment:
This test now fails on windows because the separator is os dependent.
```suggestion
    category = os.path.abspath(test_category_dir_1).split(os.sep).pop()
```

Reply:
oh I see, I was struggling with this yesterday and have no idea why fails on Windows, thank you so much for the help!